thrust/thrust/system/cuda/detail/find.h

```diff
@@ -28,6 +28,8 @@
 
 #include <thrust/detail/config.h>
 
+#include <thrust/iterator/transform_iterator.h>
+
```

Comment:
Question: are those intentionally added? If so they are at the wrong spot

Also we are missing the counting_iterator

Applies throughout

Reply:
Yeah, it seems I was sloppy with adding headers. Will fix!